Using-Postman-with-Rhetos.md

```diff
@@ -0,0 +1,63 @@
+# Using Postman with the Rhetos framework
+Create a collection for recording all requests. This will simplify and speed up testing of the application.
+First choose the authentication type - NTLM, and enter your domain data (Image 1):
+
+![Postman image 1](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image1.png)
+
+Image 2. shows the fields used for testing. Using the left dropdown menu you can select the desired HTTP method (GET, POST, PUT, PATCH, DELETE, etc.), while the center field should contain the server URL (server resource). When the desired method and URL have been entered, hit Send to send the request.
+The difference between PUT and PATCH requests is that **PUT updates the whole entity** while **PATCH only updates the targeted property** of the entity.
+
+![Postman image 2](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image2.png)
+
+If your request updates the database, then you should select *Body -> raw* and set the format to JSON (Image 3).
+
+![Postman image 3](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image3.png)
+
+The request is sent to Rhetos through a REST service in the following format: `{Rhetos server URL}/Rest/{Module}/{Entity}/`
+Image 3 shows accessing the SustavZaNabavu application through the REST service, and then accessing the module SustavZaNabavu and its entity Adresa. At the the end `/` needs to be added for accessing the resource: `http://localhost/sustavzanabavu/rest/sustavzanabavu/adresa/`
+
+All the requests are sent in the JSON format. Date properties are formatted in the following way: `\/Date(YYYY-MM-DDTHH:MM:SS.sssZ)\/` (example: `"DatumPredaje":"\/Date(2017-10-31T23:00:00.000Z)\/"`)
```

Comment:
This is not the correct format for the date properties it Rhetos REST API.
You can just remove this sentence for now. This kind of details can be explained other articles (Rhetos REST API specification).